centreon/src/Core/Media/Application/UseCase/GetMedia/GetMediaResponse.php

```
@@ -0,0 +1,69 @@
+    public string $comment;
+
+    #[Groups(['Media:Get'])]
+    public string $directory;
+
+    #[Groups(['Media:Get'])]
+    public string $filename;
+
+    #[Groups(['Media:Get'])]
+    public string $md5;
+
+    private mixed $body;
```

PHPMD (2.15.0)

Avoid unused private fields such as '$body'. (undefined)

(UnusedPrivateField)